gadgets/fsnotify/README.mdx

```diff
@@ -114,4 +114,84 @@ ig           containerd-shim  fa_resp    FANOTIFY_EVENT_TYPE_PATH_PERM   allow
 
 ## Architecture
 
-![Architecture diagram](fsnotify-architecture.svg)
+```mermaid
+sequenceDiagram
+    box rgb(245, 245, 245) Programs
+        participant Tracer
+        participant Tracee
+    end
+    box rgb(245, 245, 245) Maps
+        participant fsnotify_insert_event_ctx
+        participant enriched_fsnotify_events
+        participant fsnotify_remove_first_event_ctx
+        participant events ring buffer
+    end
+
+alt inotify
+    Note over Tracee: kprobe/inotify_handle_inode_event
+    Activate Tracee
+        rect rgb(255, 245, 168)
```

Comment:
IMO, the font color is not very visible over this background:

![image](https://github.com/user-attachments/assets/8fa5a676-df13-4f77-bf76-22e46918f11f)

Reply:
How did you render it?

On [this page](https://github.com/inspektor-gadget/inspektor-gadget/blob/3aef50b3429515d1a59446036334d44d906e3746/gadgets/fsnotify/README.mdx), I see this:

![image](https://github.com/user-attachments/assets/48ab8b96-d19d-45fa-b59e-887b4e078a1c)

Reply:
I think it's because I have dark theme on GH.